pkg/commands/git_commands/commit_file_loader.go

```diff
@@ -28,7 +28,7 @@ func (self *CommitFileLoader) GetFilesInDiff(from string, to string, reverse boo
 		Arg("--no-ext-diff").
 		Arg("--name-status").
 		Arg("-z").
-		Arg("--no-renames").
+		Arg("--no-renames"). // TODO: support renames and use rename similarity threshold
```

Comment:
I think it's intentional that we turn off rename detection here, because it makes it easier to create custom patches from added or deleted files.